pkg/consuldp/metrics_test.go

```diff
@@ -54,6 +54,7 @@ func TestMetricsServerClosed(t *testing.T) {
 }
 
 func TestMetricsServerEnabled(t *testing.T) {
+	mergedMetricsBackendBindAddr := mergedMetricsBackendBindHost + defaultMergedMetricsBackendBindPort
```

Comment:
Should we add tests that the metrics backend port can be overridden?

Reply:
Done!